spec/inverted-index-test.js

```diff
@@ -1,11 +1,12 @@
 'use strict';
+
 const books = require('./books');
 
-//A test suite to read book data
+// A test suite to read book data
 describe('Inverted Index Suite', () => {
   //Create an instance of the Index class
   const newIndex = new InvertedIndex();
-  const emptyBook = [{}];
+  const emptyBook = [];
   const sampleSentence = 'As &you can see here, we have defined *the function, useCounter(), as the target of the self-executing function %block.';
```

Comment:
What do you think about breaking `sampleSentence` as such to stay within our prescribed character limit?
>const sampleSentence = \`
    As &you can see here, we have defined *the function, useCounter(), 
    as the target of the self-executing function %block.\`;